src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.spec.ts

```diff
@@ -175,9 +175,9 @@ describe('UserFormComponent', () => {
       }
     ];
     const pwdExpirationSettings = {
-      user_pwd_expiration_warning_1: 10,
-      user_pwd_expiration_warning_2: 5,
-      user_pwd_expiration_span: 90
+      pwdExpirationSpan: 10,
+      pwdExpirationWarning1: 5,
+      pwdExpirationWarning2: 90
```

Comment:
I assume there is a reason why the value of all warnings switched, but I can't figure it out, could you enlighten me? :)